08-regression2.Rmd

```diff
@@ -424,12 +453,12 @@ coeffs
 ```
```

Comment:
In  the code chunk above, we should use the pipe again:

```
coeffs <- lm_fit %>% 
                  pull_workflow_fit() %>% 
                  tidy()
coeffs
```

Reply:
done